evaluator/evaluator.py

```diff
@@ -43,16 +44,33 @@
 
 CONSUMER_QUEUE = mqueue.MQReader(kafka_evaluator_topic)
 WEBHOOKS_QUEUE = mqueue.MQWriter(mqueue.WEBHOOKS_TOPIC)
+PAYLOAD_TRACKER_PRODUCER = mqueue.MQWriter(PAYLOAD_TRACKER_TOPIC)
 
 
 async def terminate(_, loop):
     """Trigger shutdown."""
     LOGGER.info("Signal received, stopping kafka consumers.")
     await CONSUMER_QUEUE.stop()
     await WEBHOOKS_QUEUE.stop()
+    await PAYLOAD_TRACKER_PRODUCER.stop()
     loop.stop()
 
 
+def send_msg_to_payload_tracker(msg_dict, status, status_msg=None, loop=None):
```

Comment:
move it into some common method as it was previously, pass the writer as prameter